bills-app/src/main/webapp/script.js

```diff
@@ -11,97 +11,124 @@
 
     bills.map(bill => {
         // Creating our individual html components
         let billsListItem = document.createElement('li');
         let itemTitle = document.createElement('h2');
+        let itemLink = document.createElement('a');
         let itemSector = document.createElement('p');
         let itemDate = document.createElement('p');
 
         let appendItemSector = billsListItem.appendChild(itemSector);
         let appendTitle = billsListItem.appendChild(itemTitle);
-        let appendDate = billsListItem.appendChild(itemDate);
+        let appendDate = billsListItem.appendChild(itemDate); 
+        let appendLink = billsListItem.appendChild(itemLink);
+
+        translate(bill.short_title, lang, appendTitle);
+        translate(bill.committees, lang, appendItemSector);
 
-        appendTitle.innerHTML = bill.short_title;
-        appendItemSector.innerHTML = bill.committees;
         appendDate.innerHTML = "Date introduced: " + bill.introduced_date;
+        appendLink.innerHTML = "View bill";
 
         // Giving each of our individual bill a class so we can style later
         billsListItem.setAttribute("class", "bills-list-item");
         itemTitle.setAttribute("class", "bills-list-item_title");
         itemSector.setAttribute("class", "bills-list-item_sector");
         itemDate.setAttribute("class", "bills-list-item_date");
 
+        itemLink.setAttribute("class", "bills-list-item_link");
+        itemLink.setAttribute("href", `${bill.govtrack_url}`)
+        itemLink.setAttribute("target", "_blank");
+        
         // Appending our main parent list item to our <ul>
         billsList.appendChild(billsListItem);
     });
 }
+
 function searchBillsWithApi() {
     var searchquery = document.getElementById("searchbutton").value;
 
-    if (searchquery.length != 0){
+    if (searchquery !== ""){
         var url = "https://api.propublica.org/congress/v1/bills/search.json?query=" + searchquery;
         fetch(url, {method: "GET", headers: {"X-API-Key": proPublicaApiKey}
         }).then(response => response.json()).then((searchedBills) => {
             //searchedBills is the returned json file
             var billsList = searchedBills.results[0].bills;
             clearBillData();
+
             if (billsList.length == 0) {
                 document.getElementById("bills-list").innerText = "No results found.";
             } else {
-                renderBills(billsList);
+                renderBills(billsList, "en");
             }
         });
-    } //nothing happens if the user enters nothing into the search bar
+    } else {
+        // Fetch the default selected data when search input value is empty
+        let selectedLang = localStorage.getItem('lang');
+        let selectedBillsType = localStorage.getItem('billType');
+
+        if (selectedBillsType === "recent") getRecentBills(selectedLang);
+        if (selectedBillsType === "upcoming") getUpcomingBills(selectedLang);
+    }
```

Comment:
Just a design thought: what makes sense for the user to see if they try to search bills but don't type in any query? Some options:
1. Some alert/error saying they have to type in a query
2. Automatically show some bills of some sort (as you've done here)
3. Do nothing
4. other ideas...
 
I think it'd be helpful to indicate to the user that we've noticed the query is empty and show how we're handling it, if that makes sense. It might not be super intuitive to all of a sudden see recent or upcoming bills and not quite understand how we got there. No need for changes in this PR, just wanted to bring it up!

Reply:
I think showing the user some type of response from our side letting them know that they haven't typed anything would be a good idea. I had it return to the default results since there isn't a way to go back to show all bills without actually changing the type and going back to the one you were at once you actually queried for something.

Something better can definitely be implemented, though!